include/vinecopulib/misc/tools_stl.hpp

```diff
@@ -104,6 +89,14 @@ namespace tools_stl {
         out.insert(out.end(), y.begin(), y.end());
         return out;
     }
+    
```

Comment:
Yes but [your result was sorted](http://en.cppreference.com/w/cpp/algorithm/set_symmetric_difference) as well. This makes it impossible to check whether conditioned sets in the tree representation are exactly equal to the ones implied by the matrix.